dev/requirements.txt

```diff
@@ -1,6 +1,4 @@
 click~=7.0
-jira>=2.0.0
 keyring==10.1
 gitpython
 jinja2~=2.10
-PyGithub
```

Comment:
PyGithub is used here: https://github.com/apache/airflow/blob/master/dev/airflow-github#L28

Reply:
ok - added it back in